src/plugins/embeddable/public/tests/container.test.ts

```diff
@@ -128,9 +140,20 @@ describe('container initialization', () => {
     done();
   });
 
+  it('initializes embeddables once and only once with multiple input updates', async (done) => {
+    const { container, contactCardCreateSpy } = await creatHelloWorldContainerAndEmbeddable({
```

Comment:
Looks like this has been around awhile, but I assume this should be `createHelloWorldContainerAndEmbeddable`? Nothing to fix for this PR, just thought it was a bit funny to see a typo from three years ago still sticking around 😆